common/cards/advent-of-tcg/effects/berry-bush.ts

```diff
@@ -70,20 +74,22 @@ class BerryBushEffectCard extends EffectCard {
 		}
 
 		player.hooks.afterAttack.remove(instance)
+		player.hooks.canAttach.remove(instance)
 		opponentPlayer.hooks.afterAttack.remove(instance)
 		opponentPlayer.hooks.onTurnEnd.remove(instance)
 	}
 
 	override canAttach(game: GameModel, pos: CardPosModel) {
 		const {slot} = pos
 		const {currentPlayer, opponentPlayer} = game
+		const result: CanAttachResult = []
 
-		if (!slot || slot.type !== 'hermit') return 'INVALID'
-		if (pos.player.id !== opponentPlayer.id) return 'INVALID'
-		if (opponentPlayer.board.activeRow === null) return 'INVALID'
-		if (currentPlayer.board.activeRow === null) return 'INVALID'
+		if (!slot || slot.type !== 'hermit') result.push('INVALID_SLOT')
+		if (pos.player.id !== opponentPlayer.id) result.push('INVALID_PLAYER')
+		if (opponentPlayer.board.activeRow === null) result.push('UNMET_CONDITION')
+		if (currentPlayer.board.activeRow === null) result.push('UNMET_CONDITION')
```

Comment:
There's the possibility of pushing 2 unmet conditions here, which is ok but unneeded. Better to have it add just one at maximum.